### ISSUE TITLE
SD_Connector: wrong footprint orientation

### ISSUE THREAD

**Issue by gmsotavio** [KiCad/kicad-footprints#1674]
During my inspection, I've found which several footprints in SD_Connector library have a wrong orientation. KLC F4.2 states: "Footprints should be oriented such that Pin 1 is located in the upper left corner (IPC-7351)." (http://kicad-pcb.org/libraries/klc/F4.2/). Am I right?

![Screenshot_2019-07-03_09-43-23](https://user-images.githubusercontent.com/10669626/60750392-28c1dc80-9f7e-11e9-89b8-6690e13c552b.png)
![Screenshot_2019-07-03_09-44-03](https://user-images.githubusercontent.com/10669626/60750393-295a7300-9f7e-11e9-9204-e63388662b2d.png)
![Screenshot_2019-07-03_09-44-22](https://user-images.githubusercontent.com/10669626/60750394-295a7300-9f7e-11e9-92b8-d5d4a4abec45.png)
![Screenshot_2019-07-03_09-44-49](https://user-images.githubusercontent.com/10669626/60750395-295a7300-9f7e-11e9-90a0-73801238096f.png)
![Screenshot_2019-07-03_09-45-02](https://user-images.githubusercontent.com/10669626/60750396-29f30980-9f7e-11e9-8d1b-94ab060bdd10.png)
![Screenshot_2019-07-03_09-45-13](https://user-images.githubusercontent.com/10669626/60750397-29f30980-9f7e-11e9-971f-b833276c2b28.png)
![Screenshot_2019-07-03_09-45-22](https://user-images.githubusercontent.com/10669626/60750398-2a8ba000-9f7e-11e9-9600-e5250cfb18d2.png)
![Screenshot_2019-07-03_09-45-32](https://user-images.githubusercontent.com/10669626/60750400-2a8ba000-9f7e-11e9-8869-8826aa8f7e79.png)

**Comment by evanshultz** [comment]
There are many footprints which are older and do not adhere to the latest KLC guidelines. I think these fall into that category. Thanks for pointing out these issues so they can be resolved.

If time permits, I understand Rene allows footprints to be changed at any time. However, it would certainly be nice to get this done for v6 so I've added that milestone.

**Comment by antoniovazquezblanco** [comment]
@gmsotavio will you be willing to open a PR to fix the issue?

Thanks

**Comment by gmsotavio** [comment]
@antoniovazquezblanco OK. Now that the issue has been confirmed, I will provide the fix. Thanks.

**Comment by cpresser** [comment]
Related question: Should we apply KLC 4.2 for all connectors? There are quite a lot (non SD-Card connectors) that don't follow that rule.
For example, I just looked at https://github.com/KiCad/kicad-footprints/pull/1770. 
Or "MMCX_Molex_73415-0961_Horizontal_1.0mm-PCB".
![image](https://user-images.githubusercontent.com/17516572/64464242-8a7dfe80-d107-11e9-9ed6-790d4138e06f.png)

**Comment by evanshultz** [comment]
Yes, we should. Pin 1 should normally be in the top left. It's on the left for 2-terminal footprints (not sure this applies to any connectors, but think about chip resistors). And in the top center if pin 1 isn't on a corner (like PLCC footprints).

Since this is such a minor change, I'd suggest making an issue, marking it for v6, and creating a list of connector [families] which need updating. This can be queued and then implemented closer to v6's release date. No need to make branch conflicts or change things for only a minor KLC fix right now.

**Comment by poeschlr** [comment]
Most of our connector footprints are scripted and should full fill this rule already. So i guess it will be only a few that are problematic. I agree that the first step would be assessing what needs to be done and that this would be best in an issue. After we have this information we can then decide if it makes sense to fix it now or if it is best left for later.
